TestsProject/Config/DefaultEngine.ini

```diff
@@ -98,3 +98,6 @@ ManualIPAddress=
 +CollisionChannelRedirects=(OldName="VehicleMovement",NewName="Vehicle")
 +CollisionChannelRedirects=(OldName="PawnMovement",NewName="Pawn")
 
+[/Script/UnrealEd.CookerSettings]
```

Comment:
When packaging from the editor, and checking "Enable cooking on the fly", this is the resultant change. Although from what I see when packaging from the command line, it is not cooking on the fly. It's doing the cook normally